### DISCUSSION TITLE
How are you supposed to do multi-image selection in this plugin?

### DISCUSSION THREAD

**Discussion by msqiqi10** [zanllp/infinite-image-browsing#520] [Q&A] [answered]
How are you supposed to do multi-image selection in this plugin?

**Comment by coderCK2** [comment, marked as answer]
To select multiple images one at a time, hold down `CTRL` when clicking on a single image to select it, and while continuing to hold down `CTRL` click on other single images.
To select a range of images, hold down `CTRL` when clicking on the image at the start of the range, then hold down `SHIFT` while clicking on the image at the end of the range.
